pages/linux/fdisk.md

```diff
@@ -10,3 +10,27 @@
 - Start the partition manipulator:
 
 `fdisk {{/dev/sdX}}`
+
+- Once partitioning a disk, create a partition:
```

Comment:
How about
```suggestion
- Create a partition within `fdisk`:
```
Maybe also `inside of` or `in`

Reply:
I don't like that. To me that just means, 

when you use this command, it creates a partition.

or in tldr terms

- Create a partition:

Reply:
It being interactive shows that you need a previous command to reach that stage

Reply:
That's what the `within fdisk` is for. It tells you that you have to be inside of fdisk (running it) to be able to execute the command.

Reply:
It guess, but I don't think it does as well

Reply:
```suggestion
- Once manipulating a disk, create a partition:
```
```suggestion
- Once editing a disk, create a partition:
```

Reply:
Your suggestion?

Reply:
If we're keeping that sentence structure, I'd prefer `Once partitioning a disk, ...` like it is right now.